openthread_border_router/rootfs/etc/s6-overlay/scripts/otbr-agent-configure.sh

```diff
@@ -7,3 +7,6 @@ if bashio::config.true 'nat64'; then
     bashio::log.info "Enabling NAT64."
     ot-ctl nat64 enable
 fi
+
+# Set higher transmit power
```

Comment:
Would probably make sense to add a comment here with the notes you put in the PR:
This increases link quality significantly.
`To avoid asymmetric link quality the TX power from the controller should not exceed that of what other Thread routers devices typically use.`

Reply:
Make sense, added that as a comment.